### PR TITLE
:pencil: make title and page title consistent

### PR DESCRIPTION
**Pull request by beseven**
## Related Issue
https://trello.com/c/g4dGm3SA/368-bug-explore-why-metadata-is-wrong-for-google-search-book-a-gp-appointment

## Motivation and Context
Meta description/SEO consistency. Google showing different page titles based on search term.

## Checklist

- [x] Upgraded dependencies
- [x] Tests for consistency

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

File: app/views/search-for-your-gp.nunjucks [hunk 1]
```diff
@@ -10,7 +10,7 @@
 {% endif %}
 {% endblock %}
 
-{% block pageTitle %}Find a GP to book an appointment with{% endblock %}
+{% block pageTitle %}Book a GP appointment online{% endblock %}
```
Comment (neilbmclaughlin) [inline app/views/search-for-your-gp.nunjucks]: Is it worth extracting this out to a template variable to remove the duplication?

e.g.
```
{% set title = "Book a GP appointment online" %}
{% block header %}
<h1 class="page-title">{{title}}</h1>
{% endblock %}
...
{% block pageTitle %}{{title}}{% endblock %}
...
```
Not tested this code but it gives a flavour.

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-196.dev.beta.nhschoices.net)

**Pull request merged**